### DISCUSSION TITLE
Update to Statamic 4

### DISCUSSION THREAD

**Discussion by ijpatricio** [lucky-media/landtamic#19] [General]
Hello! Is there an ETA or any plans for a Statamic 4  (upgrade)?

Thanks

**Comment by lokmanm** [comment]
Hey @ijpatricio.

The starter kit is already compatible with Statamic 4. 

Let me know if anything gets in the way.

**Comment by ijpatricio** [comment]
Thanks, @lokmanm 

I'll let you know! 🙌

Please go and update the info right away. I'm guessing you're losing business!


![image](https://github.com/lucky-media/landtamic/assets/26031459/b4b574a8-cd5b-4c96-a54e-3a95b24a1dbb)

**Comment by lokmanm** [comment]
@ijpatricio yeah we will need to update that one. Thanks!
